db/migrations/2021_05_23_000000_set_admin_password.php

```diff
@@ -43,4 +45,11 @@ public function up()
 
         $this->auth->setPassword($admin, $setupPassword);
     }
+
+    /**
+     * Reverse the migration
+     */
+    public function down(): void
+    {
```

Comment:
is here some migration needed?

Reply:
nope there isn't as it is only used to set the admin password on initial install as a shortcut for automated installs (for example when deploying to k8s)

Reply:
And also adding useless method stubs is kind of pointless?